### PR TITLE
{geo}[foss/2021b] CloudCompare v2.12.4

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/74e578232fe737c0a588c0947d5de53a for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/18f738de71da0c139753f6d501c7a108 for a full test report.

**Comment by akesandgren** [comment]
@boegelbot Please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/c/CloudCompare/CloudCompare-2.12.4-foss-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        ['lib/cloudcompare/lib%s.%s' % (x, SHLIB_EXT) for x in [
+            'CCAppCommon', 'CCCoreLib', 'CC_FBO_LIB', 'CCPluginAPI', 'QCC_DB_LIB', 'QCC_GL_LIB', 'QCC_IO_LIB'
+        ]] +
+        ['lib/cloudcompare/plugins/lib%s_PLUGIN.%s' % (x, SHLIB_EXT) for x in [
+            'QCORE_IO', 'QEDL_GL', 'QSSAO_GL', 'QADDITIONAL_IO', 'QCSV_MATRIX_IO', 'QPHOTOSCAN_IO',
+            'QBROOM', 'QCOMPASS', 'QCSF', 'QFACETS', 'QHOUGH_NORMALS', 'QHPR', 'QM3C2',
+            'QPCV', 'QRANSAC_SD', 'QSRA',
+        ]],
+    'dirs': ['share/cloudcompare/shaders'],
+}
```
Comment (smoors) [inline easybuild/easyconfigs/c/CloudCompare/CloudCompare-2.12.4-foss-2021b.eb:74]: can you add a `sanity_check_commands` here?

Comment (akesandgren) [reply easybuild/easyconfigs/c/CloudCompare/CloudCompare-2.12.4-foss-2021b.eb:74]: Can't find a useful command to test, both binaries in bin/ requires X11

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @akesandgren!

**Pull request merged**